README.md

```diff
@@ -23,19 +23,23 @@ Lifecycle Version | Platform APIs              | Buildpack APIs |
 
 ### Build
 
-* `detector` - chooses buildpacks (via `/bin/detect`)
-* `analyzer` - restores launch layer metadata from the previous build
-* `restorer` - restores cache
-* `builder` -  executes buildpacks (via `/bin/build`)
-* `exporter` - creates image and stores cache
+Either:
+* `detector` - Chooses buildpacks (via `/bin/detect`) and produces a build plan.
+* `analyzer` - Restores layer metadata from the previous image and from the cache.
+* `restorer` - Restores cached layers.
+* `builder` -  Executes buildpacks (via `/bin/build`).
+* `exporter` - Creates an image and caches layers.
+
+Or:
+* `creator` - Runs the five phases listed above in order.
```

Comment:
Do we want to add that they'll run in the same container or is it too much information for here?

Reply:
Given that the lifecycle doesn't orchestrate containers but instead runs inside a container I don't think we need to mention that here. We _should_ provide more clarity about how to use these binaries (probably replacing parts of our readme with a link to the docs instead of duplicating soon-to-be-improved docs here. For now I just want to get this one quick fix in.